### PR TITLE
C# Fundamentals : Set1 - Task6

### PR DESCRIPTION
**Pull request by tharaniannamalai**

### CODE REVIEW

File: CSharpLearning/Set1.Task6/Program.cs [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+int player1 = dice.Next(1, 7);
+int player2 = dice.Next(1, 7);
+int player3 = dice.Next(1, 7);
+int player4 = dice.Next(1, 7);
+int player5 = dice.Next(1, 7);
+
+Console.WriteLine($"Player1 Dice Roll Number : {player1}\nPlayer2 Dice Roll Number:{player2} " +
+    $"\nPlayer3 Dice Roll Number : {player3} \nPlayer4 Dice Roll Number: {player4} \nPlayer5 Dice Roll Number: {player5}");
+
+if (((player1 == player2) || (player1 == player3) || (player1 == player4) || (player1 == player5)) &&
```
Comment (ShriHumrudha) [inline CSharpLearning/Set1.Task6/Program.cs:17]: player1 == player2 is same as player2 == player1.

File: CSharpLearning/Set1.Task6/Program.cs [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+}
+
+else
+{
+    Console.WriteLine("\nNo one moved to next level.");
+}
+
+Console.ReadLine();
+
+
```
Comment (ShriHumrudha) [inline CSharpLearning/Set1.Task6/Program.cs]: Remove all the extra lines

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

File: CSharpLearning/Set1.Task6/Program.cs (hunk 1 above)
Comment (tharaniannamalai) [reply CSharpLearning/Set1.Task6/Program.cs:17]: Comment Resolved.

File: CSharpLearning/Set1.Task6/Program.cs (hunk 2 above)
Comment (tharaniannamalai) [reply CSharpLearning/Set1.Task6/Program.cs]: Comment Resolved.

File: CSharpLearning/Set1.Task6/Program.cs (hunk 1 above)
Comment (ShriHumrudha) [reply CSharpLearning/Set1.Task6/Program.cs:17]: In your modified code you are checking for player1 == player2 and player2 == player3 and player1 == player3. For the same logic, you are verifying it 3 times. There should be some simplified version, think about it.

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

**Review by ShriHumrudha — APPROVED** [review]

**Pull request merged**